### PR TITLE
ctb: Reduce heavy fuzz runs in CI

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

File: .circleci/config.yml
Comment (smartcontracts) [inline .circleci/config.yml]: Hmmm shouldn't be necessary, what was the requirement here?

**Review by smartcontracts — APPROVED** [review]

**Comment by mslipper** [comment]
Will merge once I validate that the test runtime goes down - stand by.

**Comment by mslipper** [comment]
I temporarily reverted the last contracts commit to test the runtime - will remove that change before merging.

**Pull request merged**
